Tests/ValetIntegrationTests/SinglePromptSecureEnclaveIntegrationTests.swift

```diff
@@ -20,55 +20,68 @@
 
 import Foundation
 @testable import Valet
-import LegacyValet
 import XCTest
 
+#if canImport(LocalAuthentication)
 
 class SinglePromptSecureEnclaveIntegrationTests: XCTestCase
 {
     static let identifier = Identifier(nonEmpty: "valet_testing")!
-    let valet = SinglePromptSecureEnclaveValet.valet(with: SinglePromptSecureEnclaveTests.identifier, accessControl: .userPresence)
+
+    @available(tvOS 11.0, *)
+    func valet() -> SinglePromptSecureEnclaveValet {
```

Comment:
In the PR comment you mentioned that this was done for tvOS 9 support. Is this still needed now that this API is only available on tvOS 11+?

Reply:
See this comment for clarification on why I’m making the minimum target tvOS 11: https://github.com/square/Valet/pull/209#issuecomment-576358603